libs/ui/inputs/src/lib/dropdown-selector/dropdown-selector.component.html

```diff
@@ -8,7 +11,7 @@
   </span>
   <gn-ui-button
     type="outline"
-    class="grow"
+    class="grow min-w-[165px]"
```

Comment:
Just wondering if a `whitespace-nowrap` somewhere could also do the job to prevent setting a hard value that depends on the content to look good.